Templates/DefaultProject/Template/ExportScripts/export_standalone_monolithic_engine_centric.py

```diff
@@ -0,0 +1,161 @@
+#
+# Copyright (c) Contributors to the Open 3D Engine Project.
+# For complete copyright and license terms please see the LICENSE at the root of this distribution.
+#
+# SPDX-License-Identifier: Apache-2.0 OR MIT
+#
+#
+
+
+import argparse
+import glob
+import logging
+import os
+import o3de.export_project as exp
+import pathlib
+import sys
+import time
+
+from o3de.export_project import process_command
+from o3de import manifest
+
+project_json_data = manifest.get_project_json_data(project_path=o3de_context.project_path)
+project_name = project_json_data.get('project_name')
+host_asset_platform = exp.determine_host_asset_platform()
+parser = argparse.ArgumentParser(prog='Exporter for standalone builds',
+                                 description = "Exports a project as standalone to the desired output directory with release layout. "
+                                                "In order to use this script, the engine and project must be setup and registered beforehand. ")
+parser.add_argument('-out', '--output-path', type=pathlib.Path, required=True, help='Path that describes the final resulting Release Directory path location.')
+parser.add_argument('-cfg', '--config', type=str, default='profile', choices=['release', 'profile'],
+                        help='The CMake build configuration to use when building project binaries. If tool binaries are built with this script, they will use profile mode.')
```

Comment:
Does this mean that I cannot set `release` if I built the tools in profile?

Reply:
You can set it to release. The final executables for the project will match what the chosen config is. Only tool executables (like AssetProcessor/Bundler), will be built with profile mode.